recipe/meta.yaml

```
@@ -7,7 +7,7 @@ package:
 
 source:
   url: https://pypi.io/packages/source/d/dask/dask-{{ version }}.tar.gz
```

Suggesting using hashed URL to workaround upstream issue ( https://github.com/pypi/warehouse/issues/7070 )

```suggestion
  url: https://files.pythonhosted.org/packages/68/e1/1edb5eeb15f88267075c37ee8c83c4527e462bdea328566c5b1e88db705c/dask-{{ version }}.tar.gz
```